package.json

```diff
@@ -109,6 +109,7 @@
     "chrono-node": "^1.4.8",
     "classnames": "^2.2.6",
     "d3": "^5.16.0",
+    "date-fns": "^2.16.1",
```

Comment:
Adds a bunch of date functions. I think this should be used in favor of moment from now on. Since moment has been EOL'd.